torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -811,12 +811,18 @@ class MKLDNNSubgraphSlicer {
       case aten::sigmoid:
       case aten::hardsigmoid:
       case aten::hardswish:
+      case aten::tanh:
+      case aten::batch_norm:
       // TODO: max_pool on mkldnn can be slower than in eager. ideally, we'd
       // only fuse it if we knew including max_pool lead to fewer layout
       // conversions. from initial testing including it speeds up models
       case aten::max_pool2d:
+      case aten::adaptive_max_pool2d: // not supported for floats
```

Comment:
> not supported for floats 

If it's not supported for floats, why are we fusing it?

Reply:
oh snap, the `//` should've been on the other side!